examples/provider_fallback/base.yaml

```diff
@@ -42,7 +42,9 @@ spec:
               value: us.meta.llama3-2-1b-instruct-v1:0
       backendRefs:
         - name: provider-fallback-always-failing-upstream  # This is the primary backend and trying to speak TLS, which always fails.
+          priority: 0
         - name: provider-fallback-aws
+          priority: 1
```

Comment:
Can you also delete the `fallback: true` below as it's overridden by this

Reply:
Yes I deleted the fallback flag below